### PR TITLE
feat(content): create links for editing

### PR DESCRIPTION
**Pull request by petrjasek**
so that user can open items in different browser tabs

### CODE REVIEW

File: app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -813,15 +813,14 @@
 	            })
 	            .activity('edit.text', {
 	            	label: gettext('Edit item'),
+                    href: '/authoring/:_id',
```
Comment (amagdas) [inline app/scripts/superdesk-authoring/authoring.js]: This needs to be added for packages as well...

**Comment by amagdas** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Comment by eozan** [comment]
:+1:

**Pull request merged**